addons/addon-environment-sc-api/packages/environment-type-mgmt-services/lib/environment-type/env-type-candidate-service.js

```diff
@@ -220,7 +224,7 @@ class EnvTypeCandidateService extends Service {
         );
       }),
     );
-    return productEnvTypes;
+    return _.filter(productEnvTypes);
```

Comment:
Nit pick: specify it's filtering out null here?

Reply:
It's fairly common to use filter in this way to filter out "falsy" values. I updated it anyways to add an explicit predicate